ledger_device_sdk/src/ecc.rs

```diff
@@ -377,7 +377,12 @@ impl<const P: usize> ECPublicKey<P, 'E'> {
 ///
 /// Checks consistency of curve choice and key length
 /// in order to prevent the underlying syscall from throwing
```

Comment:
Note that the syscall can throw if the requested curve or path haven't been declared in the manifest.

Reply:
`no_throw` version is not available through bindings as it is defined as a static inline function in C SDK.

Reply:
My remark was just about the fact that the comment isn't perfectly true... There is no way to avoid the syscall to throw.